### DISCUSSION TITLE
Creating Teams

### DISCUSSION THREAD

**Discussion by merkourisa** [merkourisa/Digital-Storytelling-Group-Assignment#1] [Announcements]
**To create your team (3-6 fellow students) follow the above steps:** _Deadline: 31/03/2023_

1. After you have created your Github account make a new comment by inserting the above table in your comment. In this way, you can create a new team and invite new members. 
2. If you would like to join a team you can ask to join by making the appropriate comment. If there is no team available you must create a new one.
3. Every time there is a change in the team (new member) you have to update the teams details by editing the comment, so it's always visible which teams are active and which are their members.

Name of the team: XXX
GitHub url: [XXX](url)
Accepting new member: Yes/No

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | dpsdXXXXX | gh link |
| 2 | dpsdYYYYY | gh link |
| 3 | ... | ... | 
| 4 | ... | ... | 
| 5 | ... | ... | 
| 6 | ... | ... | 

This discussion is for organizing the teams. For any other question, you can open a new discussion.

**Comment by KostisG** [comment]
Name of the team : DreamTeam
GitHub url : https://github.com/KostisG/Digital-Storytelling-Group-Assignment
Accepting new members : No 

| # | AM | GitHub profile |
|       :---:         |     :---:      |     :---:      |
| 1  | Dpsd19003    | https://github.com/AnastasiaAmp    |
| 2  | Dpsd19025  | https://github.com/KostisG    |
| 3  | Dpsd19047    | https://github.com/RaniaKrz|
| 4  | Dpsd19138      | https://github.com/BYKORDOU    |

**Reply by merkourisa** [reply]
@KostisG δεν έχεις ανεβάσει στο e-class το ζητούμενο pdf με τα στοιχεία των μελών της ομάδας σου

**Reply by KostisG** [reply]
Έχετε δίκιο το είχα ξεχάσει το ανέβασα τώρα

**Reply by merkourisa** [reply]
@KostisG approved

**Comment by NemaZyst** [comment]
Name of the team: Triumvirate
GitHub url: [https://github.com/NemaZyst/Digital-Storytelling-Group-Assignment](url)
Accepting new member: No

<html>
<body>


# | ΑΜ | GitHub profile
-- | -- | --
1 | dpsd18062 | [https://github.com/NemaZyst](url)
2 | dpsd18016 | [https://github.com/arizona515](url)
3 | dpsd19083 | [https://github.com/Tmbtim](url)


</body>
</html>

**Reply by itsdjdannysy** [reply]
Παιδιά καλησπέρα είμαστε 2 άτομα και ψάχνουμε ομάδα. Μπορούμε να μπούμε μαζί σας; dpsd19124 & dpsd18119

**Reply by merkourisa** [reply]
@NemaZyst στο e-class έχεις ανεβάσει το design document σε μορφή pdf αντί για το Transmedia document σε μορφή markdown (README.md). Δεν υπάρχει επίσης τo link του διαχειριστή για το repository της ομαδικής. [Διάβασε τις οδηγίες](https://github.com/merkourisa/Digital-Storytelling-Group-Assignment)

**Reply by NemaZyst** [reply]
Το λινκ του repository ειναι https://github.com/NemaZyst/Digital-Storytelling-Group-Assignment

**Reply by NemaZyst** [reply]
Όσον αφορά το design document επειδή είμαι εκτός Σύρου θα το ανανεώσω σε μορφή md  όταν επιστρέψω αλλά μπορείτε να το προβάλλετε κανονικά.

**Comment by dpsd18023** [comment]
**Name of the team:**  TheAnimators
**GitHub url:** [github.com/dpsd18023/Digital-Storytelling-Group-Assignment](https://github.com/dpsd18023/Digital-Storytelling-Group-Assignment)
**Accepting new member:** No

| # | AM | GitHub profile |
|       :---:         |     :---:      |     :---:      |
| 1  | Dpsd16028    |   https://github.com/GDanisDpsd16028  |
| 2  | Dpsd17064    |   https://github.com/aggelosman  |
| 3  | Dpsd17073    |   https://github.com/Prokopisbelias  |
| 4  | Dpsd18023    |   https://github.com/dpsd18023  |
| 5  | Dpsd18108    |   https://github.com/dpsd18108  |
| 6  | Dpsd18128    |   https://github.com/aggelosFyl   |

**Reply by dpsd14073** [reply]
Καλήσπέρα @dpsd18023, μπόρω να μπώ στην όμαδα σας? dpsd14073 , https://github.com/dpsd14073

**Reply by Prokopisbelias** [reply]
Καλησπέρα είμαστε δύο άτομα και ενδιαφερόμαστε να μπούμε στην ομάδα
Dpsd17073 Dpsd17064

**Reply by merkourisa** [reply]
@dpsd18023 δεν έχεις ανεβάσει στο e-class το ζητούμενο pdf με τα στοιχεία των μελών της ομάδας σου

**Reply by merkourisa** [reply]
@Prokopisbelias μήπως να δημιουργούσατε μια ομάδα μαζί με @dpsd14073

**Comment by Feedmepeople** [comment]
Name of the team: Rock and Stone
GitHub url: https://github.com/Feedmepeople/Digital-Storytelling-Group-Assignment
Accepting new member: No

---|-----**AM**-----|-------**GitHub_Profile**----------------------|
1.  | dpsd18004 | https://github.com/Feedmepeople
2.  | dpsd18078 | https://github.com/jasonbizas
3.  | dpsd18029 | https://github.com/dpsd18029

**Reply by Prokopisbelias** [reply]
Καλησπέρα είμαστε δύο άτομα και ενδιαφερόμαστε να μπούμε στην ομάδα 
Dpsd17073    Dpsd17064

**Reply by merkourisa** [reply]
@Feedmepeople approved

**Reply by Feedmepeople** [reply]
@merkourisa thanks teach

**Reply by Feedmepeople** [reply]
@merkourisa Σχετικά με το παράδειγμα του pp που μας δίνετε στην εκφώνηση, δε μου το εμφανίζει. Μου βγάζει σφάλμα και με επιστρέφει στο αρχικό branch. Το έχετε κάπου στο eclass να το δω;

**Comment by ThanosSl** [comment]
**Name of the team:** Last of Ass 
**GitHub url:** https://github.com/ThanosSl/Digital-Storytelling-Group-Assignment
**Accepting new member:** No

<kbd>

---|-----AM-----|-------GitHub_Profile------|
1.  dpsd19120 |  https://github.com/ThanosSl
2.  dpsd19019 |  https://github.com/dpsd19019
3.  dpsd19113 |  https://github.com/dpsd19113

</kbd>

**Reply by merkourisa** [reply]
@ThanosSl approved

**Comment by Stathenia** [comment]
Name of the team: The Weakest Links
GitHub url: https://github.com/Stathenia/Digital-Storytelling-Group-Assignment
Accepting new member: ΝΟ
| # | ΑΜ | GitHub profile|
| ------------- | ------------- | ------------- |
| 1  | dpsd19103 | [gh link](https://github.com/Stathenia)|
| 2  | dpsd19014 | [gh link](https://github.com/ZoiGalanidpsd19014) |
| 3  | dpsd19071 | [gh link](https://github.com/GeorgeDPSD)|
| 4  | dpsd19068  | [gh link](https://github.com/ElenaLyra) |
| 5  | dpsd19067 | [gh link](https://github.com/dimitrisl11) |

**Reply by merkourisa** [reply]
@Stathenia approved

**Comment by dpsd19061** [comment]
Name of the team: veruca
GitHub url: [Repository](https://github.com/dpsd19061/Digital-Storytelling-Group-Assignment)
Accepting new member: No
| # | AM | GitHub profile |
| ------------- | ------------- | ------------- |
| 1  | dpsd19032 | [gh link](https://github.com/evadost?fbclid=IwAR1GgGbCq7M5_pmyHbNITaL3ElYGhxIJ2ZUUTVU2CovpRYpkvYpNe92mw6E) |
| 2  | dpsd19044 | [gh link](https://github.com/dpsd19044?fbclid=IwAR3t5F6R6p6gnOyp_6_MbvnKHEFY9FoW2eOPoXWkYu6PvYota-JwCZle07I) |
| 3  | dpsd19061 | [gh link](https://github.com/dpsd19061) |
| 4  | dpsd19080 | [gh link](https://github.com/dpsd19080) |

**Reply by itsdjdannysy** [reply]
Παιδιά καλησπέρα είμαστε 2 άτομα και ψάχνουμε ομάδα. Μπορούμε να μπούμε μαζί σας; dpsd19124 & dpsd18119

**Reply by merkourisa** [reply]
@dpsd19061 approved

**Comment by Ebabouraki** [comment]
Name of the team: Καλιάτσα
GitHub url: [Repository](https://github.com/Ebabouraki)
Accepting new member: No
| # | AM | GitHub profile |
| ------------- | ------------- | ------------- |
| 1  | dpsd19009 | [gh link](https://github.com/Ebabouraki) |
| 2  | dpsd19049 | [gh link](https://github.com/dpsd19049) |
| 3  | dpsd19126 | [gh link](https://github.com/dpsd19126) |

**Reply by merkourisa** [reply]
@Ebabouraki approved

**Comment by itsdjdannysy** [comment]
Name of the team: Anim8
GitHub url: https://github.com/itsdjdannysy/Digital-Storytelling-Group-Assignment
Accepting new member: yes

        AM                       Github profile 
1.     dpsd19124        https://github.com/itsdjdannysy
2.     dpsd18119        https://github.com/Spitfire1221

**Comment by AnnaMitsiani** [comment]
</body>
</html>Name of the team: Hi-5

GitHub url:https://github.com/Sophiapapadopoulou
Accepting new member: No

#	ΑΜ	GitHub profile
1	dpsd18098	https://github.com/dpsd18098
2	dpsd19095	https://github.com/Sophiapapadopoulou
3	dpsd18003	https://github.com/dpsd18003
4	dpsd19082	https://github.com/dpsd19082
5	dpsd19074	https://github.com/AnnaMitsiani

**Reply by merkourisa** [reply]
@AnnaMitsiani approved

**Comment by kyriaki-kyriakidou** [comment]
Name of the team:  Sunday rose
GitHub url: https://github.com/kyriaki-kyriakidou/Digital-Storytelling-Group-Assignment
Accepting new member: No

    AM       |         GitHub_Profile                                                
   dpsd20056 | https://github.com/kyriaki-kyriakidou                                
   dpsd20131 | https://github.com/Evangelia-Triantafyllia-Chatzichristofi

**Reply by itsdjdannysy** [reply]
Παιδιά καλησπέρα είμαστε 2 άτομα και ψάχνουμε ομάδα. Μπορούμε να μπούμε μαζί σας; dpsd19124 & dpsd18119

**Reply by merkourisa** [reply]
@kyriaki-kyriakidou approved

**Comment by quarufus** [comment]
Name of the group: Untitled #5
Github url: https://github.com/quarufus/Digital-Storytelling-Group-Assignment

|#|AM|Github profile|
|-|-|-|
|1|dpsd17085|https://github.com/quarufus|
|2|dpsd17067|https://github.com/Nikf0r0s|
|3|dpsd19093|https://github.com/papadimitriou19093|
|4|dpsd19037|https://github.com/dpsd19037|
|5|dpsd18109|htpps://github.com/dpsd18109|

**Reply by merkourisa** [reply]
@quarufus δεν έχεις ανεβάσει στο e-class το ζητούμενο pdf με τα στοιχεία των μελών της ομάδας σου

**Reply by quarufus** [reply]
Καλησπέρα, συγγνώμη για την καθηστέρηση αλλά δεν καταλάβαμε ότι πρέπει να το ανεβάσουμε και στο eclass.

**Comment by dpsd18133** [comment]
Name of the team: Two for tea
GitHub url: https://github.com/dpsd18133/Digital-Storytelling-Group-Assignment
Accepting new member: No

#	ΑΜ	GitHub profile
1.	dpsd18133	https://github.com/dpsd18133 
2.	dpsd18045	https://github.com/dpsd18045

**Reply by merkourisa** [reply]
@dpsd18133 approved

**Comment by Marina-Eleni-Anastasiadou** [comment]
Name of the Team: The burned cells
Github url :https://github.com/Marina-Eleni-Anastasiadou/Digital-Storytelling-Group-Assignment


| AM| GITHUB PROFILES |
|-----:|-----------|
|  dpsd19005| https://github.com/Marina-Eleni-Anastasiadou|
|  dpsd19140| https://github.com/DPSD19140   |

**Reply by merkourisa** [reply]
@Marina-Eleni-Anastasiadou approved

**Comment by MariaSiafali117** [comment]
> Name of the team : Power Puff Girls  🦸‍♀️
GitHub url : [PROFILE](https://github.com/MariaSiafali117)
Accepting new member: : No

| # | AM | Github Profile | 
| :---         |     :---:      |       ---: | 
|1   | dpsd19109   |[PROFILE](https://github.com/StellaPiperaki)  | 
| 2   | dpsd19036  |[PROFILE](https://github.com/dpsd19036)  |  
| 3   | dpsd19117  |[PROFILE](https://github.com/MariaSiafali117) |

**Reply by merkourisa** [reply]
@MariaSiafali117 approved

**Comment by tonygavridakis** [comment]
</body>
</html>Name of the team: ΠΑΣΟΚ 
GitHub url: (https://github.com/sdoulaveris/Digital-Storytelling-Group-Assignment)
Accepting new member: No

#	ΑΜ	GitHub profile
1	dpsd19033	[gh link](https://github.com/sdoulaveris)
2	dpsd19013	[gh link](https://github.com/tonygavridakis)
3	dpsd19041	[gh link](https://github.com/lydiaznt)
4	dpsd19079	[gh link](https://github.com/BizremisA)

**Reply by merkourisa** [reply]
@tonygavridakis δεν έχεις ανεβάσει στο e-class το ζητούμενο pdf με τα στοιχεία των μελών της ομάδας σου

**Comment by Costas-Lemonis** [comment]
Name of the team:Comedy of Errors
GitHub url: [Profile](https://github.com/Costas-Lemonis/Digital-Storytelling-Group-Assignment)
Accepting new member: No

|#| ΑΜ|GitHub profile|
|---|---|---|
|1| dpsd19055| [Profile](https://github.com/Giannis-Kokkinis)|
|2 | dpsd19065| [Profile](https://github.com/Costas-Lemonis)|
|3| dpsd19097| [Profile](https://github.com/dpsd19097)|
|4| dpsd19121| [Profile](https://github.com/mar-stamp)|
|5| dpsd22901| [Profile](https://github.com/LauraCLP)|
|6| dpsd18132| [Profile](https://github.com/Chatzilyras-Michalis)|

**Reply by merkourisa** [reply]
@Costas-Lemonis approved

**Comment by dpsd17031** [comment]
Name of the team: Boomer Talk
GitHub url: [https://github.com/dpsd17031/Digital-Storytelling-Group-Assignment/tree/patch-1](https://github.com/dpsd17031/Digital-Storytelling-Group-Assignment/tree/patch-1?fbclid=IwAR254TPHx2v3sfvL4iXNmHGc3nvOQw1mmdzUkQkZfB69eyGLK-5n_RDVPFk)
Accepting new member: No
# | AM | GitHub profile
-- | -- | --
1 | Dpsd17072 | https://github.com/YBekas
2 | Dpsd17016 | https://github.com/dpsd17016
3 | Dpsd17031 | https://github.com/dpsd17031
4 | Dpsd17120 | https://github.com/dpsd17120

**Reply by merkourisa** [reply]
@dpsd17031 δεν έχεις ανεβάσει στο e-class το ζητούμενο pdf με τα στοιχεία των μελών της ομάδας σου

**Comment by Zantzou** [comment]
team name : StoryYellers
accepting new member : no
| # | am | githubLink |
| ---: | ---: | ---: |
| 1 | dpsd18140 | https://github.com/Zantzou |
| 2 | dpsd18130 |https://github.com/dpsd18130 | 
| 3 | dpsd19092 | https://github.com/dpsd19092 |
| 4 | dpsd19122 |  https://github.com/athenast |
| 5 | dpsd19012 | https://github.com/desp5 |

**Reply by merkourisa** [reply]
@Zantzou δεν έχεις ανεβάσει στο e-class το ζητούμενο pdf με τα στοιχεία των μελών της ομάδας σου. Επίσης μήπως να δεχόσασταν στην ομάδα σας 2 ακόμα μέλη -> dpsd19124 & dpsd18119 (@itsdjdannysy)

**Reply by dpsd14073** [reply]
Καλησπέρα dpsd18140, μπορώ να γίνω μέλος? dpsd14073 ,αλλά βρίσκομαι Αθήνα αν δεν είναι πρόβλημα.
